x11-misc/adriconf/adriconf-2.4.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop
+
+DESCRIPTION="Advanced DRI Configurator"
+HOMEPAGE="https://gitlab.freedesktop.org/mesa/adriconf/"
+if [[ "${PV}" == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/mesa/adriconf.git"
+else
+	MY_P="${PN}-v${PV}"
+	SRC_URI="https://gitlab.freedesktop.org/mesa/${PN}/-/archive/v${PV}/${MY_P}.tar.bz2 -> ${P}.tar.bz2"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${MY_P}"
+fi
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE="wayland"
```

Comment:
This USE flag does nothing - did you intend to put some configure option for it?